composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tokenio/sdk",
   "description": "PHP SDK for interacting with the Token System",
-  "version": "1.0.0-beta-4",
+  "version": "1.1.0",
```

Comment:
1.0.0?

Reply:
it already exists https://github.com/tokenio/sdk-php/commit/446d8b2b68ab3a7adf675e15f96ac407c668fded

Reply:
never published